### ISSUE TITLE
Testing with the vanilla client

### ISSUE THREAD

**Issue by rom1504** [mhsjlw/flying-squid#174]
https://github.com/PrismarineJS/node-minecraft-protocol/issues/281

Making sure it doesn't make the server crash would be a good start.

**Comment by javacoolme** [comment]
Crashing for me when a client connects:

```
....../flying-squid/dist/lib/plugins/login.js:79
                throw _context.t0;
                ^

TypeError: Cannot read property 'header' of undefined
    at Object.module.exports.player (....../flying-squid/dist/lib/plugins/header.js:17:59)
    at ....../flying-squid/dist/lib/plugins/login.js:62:44
    at Array.forEach (native)
    at Object._callee$ (....../flying-squid/dist/lib/plugins/login.js:61:18)
    at tryCatch (....../flying-squid/node_modules/regenerator-runtime/runtime.js:65:40)
    at Generator.invoke [as _invoke] (....../flying-squid/node_modules/regenerator-runtime/runtime.js:303:22)
    at Generator.prototype.(anonymous function) [as next] (....../flying-squid/node_modules/regenerator-runtime/runtime.js:117:21)
    at step (....../flying-squid/node_modules/babel-runtime/helpers/asyncToGenerator.js:17:30)
    at ....../flying-squid/node_modules/babel-runtime/helpers/asyncToGenerator.js:35:14
    at Promise (<anonymous>)
    at F (....../flying-squid/node_modules/core-js/library/modules/_export.js:35:28)
    at Server.<anonymous> (....../flying-squid/node_modules/babel-runtime/helpers/asyncToGenerator.js:14:12)
    at Server.<anonymous> (....../flying-squid/dist/lib/plugins/login.js:91:19)
    at emitOne (events.js:115:13)
    at Server.emit (events.js:210:7)
    at loginClient (....../flying-squid/node_modules/minecraft-protocol/src/server/login.js:134:12)
    at ....../flying-squid/node_modules/minecraft-protocol/src/server/login.js:91:9
    at Request._callback (....../flying-squid/node_modules/yggdrasil/lib/Server.js:50:7)
    at Request.self.callback (....../flying-squid/node_modules/request/request.js:188:22)
    at emitTwo (events.js:125:13)
    at Request.emit (events.js:213:7)
    at Request.<anonymous> (....../flying-squid/node_modules/request/request.js:1171:10)
```

**Comment by rom1504** [comment]
are you running the code at #286 ?

**Comment by javacoolme** [comment]
@rom1504 I think so, let me try again...

**Comment by javacoolme** [comment]
@rom1504 Apparently not, thanks!
